src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -132,7 +132,8 @@ public void singleFromStringReturnsEmptyIfNoEntryRecognized() throws IOException
 
     @Test
     public void parseRecognizesEntry() throws IOException {
-        ParserResult result = new BibtexParser(importFormatPreferences).parse(new StringReader("@article{test,author={Ed von Test}}"));
+        ParserResult result = new BibtexParser(importFormatPreferences)
```

Comment:
Well, I changed about 100 or so calls marked deprecated by someone else (as far as I can tell, very few of the similar calls were actually replaced by whoever marked it as deprecated, not even in the actual code). There are still things to clean up before the methods are removed and all tests there can make use of it.